pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -1,29 +1,25 @@
 # -*- coding: utf-8 -*-
 from pokemongo_bot.event_manager import EventHandler
 from pokemongo_bot.base_dir import _base_dir
-import json
-import os
 import time
 import telegram
 import thread
 import re
-from pokemongo_bot.datastore import Datastore
 import pprint
+from pokemongo_bot.datastore import Datastore
 from pokemongo_bot import inventory
-
+from telegram.utils import request
 
 DEBUG_ON = False
 
-class FileIOException(Exception):
-    pass
-
 class TelegramClass:
 
     update_id = None
-
+    session = None
```

Comment:
Nothing at all. Was part of something I was testing, now unneccessary. Removing before merging.